### ISSUE TITLE
Full screen?

### ISSUE THREAD

**Issue by Elmagob99** [FluBBaOfWard/S8DS#11]
I use this emulator on my Dsi with twilightmenu++ and i was wondering if there is a chance to put the games on full screen, the aspect options doesnt change too much on the screen size, there will be an option in the future? thanks

**Comment by FluBBaOfWard** [comment]
I guess you're talking about GameGear games and not Master System games? Or do you mean that SMS games that use the 224 height mode should cover both DS screens? 

GG games won't be scaled, sorry.

**Comment by Elmagob99** [comment]
> I guess you're talking about GameGear games and not Master System games? Or do you mean that SMS games that use the 224 height mode should cover both DS screens? GG games won't be scaled, sorry.

yes i mean GG games, i want to play Sylvan tale on my 2ds but the screen is too small and i wanted to stretch it to full screen at least

**Comment by seanarim0** [comment]
I've tested this on a 3DS XL.
When playing GG even if you select Display Options->Scale to Fit, it looks the same as Scale 1:1. 
Is there any workaround on our end to get it to scale fully to the display height?

S8DS V1.1.7 2023-11-07 
ARMZ80 V1.9.9
SEGAVDP V0.7.1
ARMSNGG76496 V1.6.1
ARMAY38910 V0.6.0
ARMSCC V0.2.1

**Comment by Elmagob99** [comment]
> I've tested this on a 3DS XL. When playing GG even if you select Display Options->Scale to Fit, it looks the same as Scale 1:1. Is there any workaround on our end to get it to scale fully to the display height?
> 
> S8DS V1.1.7 2023-11-07 ARMZ80 V1.9.9 SEGAVDP V0.7.1 ARMSNGG76496 V1.6.1 ARMAY38910 V0.6.0 ARMSCC V0.2.1

according to Flubba no, i gave up on playing stretched gg games lol

**Comment by mrstux** [comment]
I implemented a Game Gear Upscaler feature. 

Next Best thing after actually having a GG ;)

https://github.com/FluBBaOfWard/S8DS/pull/24
